components/pedigree/resources/src/main/resources/pedigree/view/nodeMenu.js

```diff
@@ -702,12 +702,14 @@ define([
                 var result = this._generateEmptyField(data);
                 var cancerList = editor.getCancerLegend()._getAllSupportedCancers();
 
-                var div = new Element('div', {'class': 'cancer_field cancer-header field-no-user-select'} );
-                var label1 = new Element('label', {'class': 'cancer_label_field'} ).update("Name");
-                var label2 = new Element('label', {'class': 'cancer_status_select'} ).update("Status");
-                var label3 = new Element('label', {'class': 'cancer_age_select'} ).update("As of");
+                var div = new Element('tr', {'class': 'cancer_field cancer-header field-no-user-select'} );
+                var label1 = new Element('td').insert(new Element('label', {'class': 'cancer_label_field'} ).update("Name"));
+                var label2 = new Element('td').insert(new Element('label', {'class': 'cancer_status_select'} ).update("Status"));
+                var label3 = new Element('td').insert(new Element('label', {'class': 'cancer_age_select'} ).update("As of"));
                 div.insert(label1).insert(label2).insert(label3);
-                result.inputsContainer.insert(div);
+                result.inputsContainer.table = new Element('table');
```

Comment:
can just use var table = .... and then do one result.inputContainer.insert(table) at the very very end once tables is completely composed.